src/actions/user.actions.js

```diff
@@ -38,14 +38,10 @@ export const getUserInfoWithToken = ({ token }: { token: string }) =>
     }
   });
 
-export const saveTokenToLocalStorage = ({
-  token
-}: {
-  token: string | null
-}) => ({
+export const saveTokenToLocalStorage = (user = { token: null }) => ({
```

Comment:
This isn't what I meant, There is no value in the user object here. the only thing you care about is the token.
Which means you can pass the token as a single argument and the payload would be just `token` without `user` being mentioned at all.

This would be different if you wanted to persist the entire user object and the action would be called saveUserToLocalStorage

Reply:
This is still incorrect, it should take the token only

Reply:
Fixed